upstream/CVE-2024-3177.json

```diff
@@ -0,0 +1,52 @@
+{
+  "id": "CVE-2024-3177",
+  "modified": "2024-04-17T20:53:58Z",
+  "published": "2018-04-16T20:53:58Z",
+  "summary": "Bypassing mountable secrets policy imposed by the ServiceAccount admission plugin",
+  "details": "A security issue was discovered in Kubernetes where users may be able to launch containers that bypass the mountable secrets policy enforced by the ServiceAccount admission plugin when using containers, init containers, and ephemeral containers with the envFrom field populated. The policy ensures pods running with a service account may only reference secrets specified in the service account’s secrets field. Kubernetes clusters are only affected if the ServiceAccount admission plugin and the kubernetes.io/enforce-mountable-secrets annotation are used together with containers, init containers, and ephemeral containers with the envFrom field populated.",
+  "affected": [
+    {
+      "package": {
+        "ecosystem": "kubernetes",
+        "name": "k8s.io/apiserver"
```

Comment:
PRs don't contain changes for `api` package.
Does `k8s.io/apiserver` use [plugin/pkg/admission/serviceaccount/admission.go](https://github.com/kubernetes/kubernetes/pull/124322/files#diff-7189151f98b7e47974de2ccbd5bd266e0326bef6eaa94ccdf7408c2085bc8df8)?

Reply:
yes I think so:
yes I think so:
https://github.com/ritazh/kubernetes/blob/ad9b60e2c9ddb21e8b00cabbe27e639638a0ea88/pkg/kubeapiserver/options/plugins.go#L51